blockchain/txvalidator.go

```diff
@@ -323,6 +328,19 @@ func (b *BlockChain) checkVoteOutputs(blockHeight uint32, outputs []*Output, ref
 	return nil
 }
 
+func (b *BlockChain) checkCRImpeachmentContent(blockHeight uint32, content outputpayload.VoteContent) error {
+	if b.crCommittee.IsInVotingPeriod(blockHeight) {
```

Comment:
should remove this because voting period and current CR committee duty period can overlap

Reply:
Fixed